lamin_cli/compute/README.md

```diff
@@ -0,0 +1,204 @@
+## Lamin Compute Development
+
+## Getting started
+
+1. Create a local script
+
+save your script: `helloworld_lamin.py`
+
+```python
+import os
+import lamindb as ln
+
+API_KEY = os.environ['lamin_user_api_key']
+PROJECT_NAME = os.environ['lamin_project_name']
+INSTANCE_NAME = os.environ['lamin_instance_name']
+INSTANCE_OWNER = os.environ['lamin_instance_owner']
+
+# LAMIN SETUP
+ln.setup.login(api_key=API_KEY)
+ln.connect(f'{INSTANCE_OWNER}/{INSTANCE_NAME}')
+my_project = ln.Project(name=PROJECT_NAME).save()
+ln.track(project=PROJECT_NAME)
+
+def say_hello():
+    print('Hello, World! lamin, user key has been passed successfully')
+
+
+if __name__ == '__main__':
+    say_hello()
+
+ln.finish()
+```
+
+2. Run your script from Lamin CLI
+
+```
+lamin run ./helloworld_lamin.py --project modal_project
+```
+
+##### Note: once `--project` is specified the environment is attached to it and will not need to be rebuilt.
+
+#### Running a job with a premade image and additional python dependancies
+
+1. Create a local script
+
+save your script locally `helloworld_gpu.py`
+
+```python
+import os
+import lamindb as ln
+
+API_KEY = os.environ['lamin_user_api_key']
+PROJECT_NAME = os.environ['lamin_project_name']
+INSTANCE_NAME = os.environ['lamin_instance_name']
+INSTANCE_OWNER = os.environ['lamin_instance_owner']
+
+# LAMIN SETUP
+ln.setup.login(api_key=API_KEY)
+ln.connect(f'{INSTANCE_OWNER}/{INSTANCE_NAME}')
+my_project = ln.Project(name=PROJECT_NAME).save()
+ln.track(project=PROJECT_NAME)
+
+def say_hello():
+    import torch
+    print('Imported pytorch and detecting GPUs')
+    print(torch.cuda.is_available())
+
+
+if __name__ == '__main__':
+    say_hello()
+
+ln.finish()
+
+
+```
+
+We will use a public image `nvcr.io/nvidia/pytorch:22.12-py3`
```

Comment:
Can we use a more recent release, please?

https://docs.nvidia.com/deeplearning/frameworks/pytorch-release-notes/rel-25-03.html